src/main.js

```diff
@@ -213,15 +213,24 @@ let load = Loader({
     }
   },
 
-  'expire-sentry': {
-    requires: ['cfg', 'sentryManager'],
-    setup: async ({cfg, sentryManager}) => {
+  'expire': {
```

Comment:
I would have made two processes, but I see how both these do a few things... Perhaps we can run them in parallel...


Note: there is a value in having two processes when writing tests.